src/components/Identity/getEcid/createGetEcid.js

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import createIdentityPayload from "./createIdentityPayload";
+
+const identityPayload = createIdentityPayload();
```

Comment:
We should be creating a new payload for each request. Without doing so, this same payload object could be used across requests and even across instances, which could be really bad. We don't know (or shouldn't assume) what kind of payload mutations may be sticking around from one request to the next.